integration-tests/kotlin-serialization/pom.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This test also needs to be added to https://github.com/quarkusio/quarkus/blob/main/.github/native-tests.json

Reply:
i always forget this one.  thanks.